### ISSUE TITLE
Better to move status to the model or a separate constants file.

### ISSUE THREAD

**Issue by priteshgudge** [jadhavmanoj/agrostar-asset#1]
https://github.com/jadhavmanoj/agrostar-asset/blob/1d771913ec55fbe5fe754aa18daf59f5cc484200/functionality/asset.py#L70

**Comment by jadhavmanoj** [comment]
Yes, That should be the best way to write code. I forget to add this in constant. 
Thanks @priteshgudge
